management/tests/mocks/test-server/routes/update-identity.ts

```diff
@@ -0,0 +1,169 @@
+import express from "express";
```

Comment:
I wonder if using a bunch of `if`s instead of `if else` would make the reading better. What do you think?

Reply:
I don't really know what I prefer. One or the other way should work the same, so it's just a matter of reading preferences I guess. I will stand with the team's majority opinion on this one 🙂

Reply:
Ok, I vote for `if`s then, make this big block breath a bit 😄 .

Reply:
I vote for `if else` because it often avoids some weird things like: 
> I'm going in this `if` because you didn't think about this edge case scenario